### DISCUSSION TITLE
Make a convenient button for adding a site on the main page

### DISCUSSION THREAD

**Discussion by OmlineEditor** [bromite/bromite#2185] [Ideas]
make a convenient button for adding a site.
it is necessary to make it as convenient and understandable as possible so that those who use it intuitively understand how to add a website.

![Screenshot_2](https://user-images.githubusercontent.com/13348933/178296021-fafdd252-6094-497d-a29c-1a9e02aeb66b.jpg)

**Comment by csagan5** [comment]
This has nothing to do with privacy, you can ask for this feature on the Chromium issue tracker.
